Building DAG of jobs...
Provided cores: 2
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	2	metrics
	3

rule metrics:
    input: data/sorted_reads/combined_HyperPrep.bam, data/samples/combined/combined_HyperPrep.sam
    output: metrics/combined_HyperPrep.bam.csv
    jobid: 6
    wildcards: sample=HyperPrep

rule metrics:
    input: data/sorted_reads/combined_HyperPlus.bam, data/samples/combined/combined_HyperPlus.sam
    output: metrics/combined_HyperPlus.bam.csv
    jobid: 2
    wildcards: sample=HyperPlus

Error in rule metrics:
    jobid: 2
    output: metrics/combined_HyperPlus.bam.csv

RuleException:
CalledProcessError in line 86 of /home/dewald/Documents/snakemake/Snakefile:
Command ' set -euo pipefail;  /usr/bin/python3 /home/dewald/Documents/snakemake/scripts/.snakemake.ib_f1tma.metrics.py ' returned non-zero exit status 1.
  File "/home/dewald/Documents/snakemake/Snakefile", line 86, in __rule_metrics
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Will exit after finishing currently running jobs.
Error in rule metrics:
    jobid: 6
    output: metrics/combined_HyperPrep.bam.csv

RuleException:
CalledProcessError in line 86 of /home/dewald/Documents/snakemake/Snakefile:
Command ' set -euo pipefail;  /usr/bin/python3 /home/dewald/Documents/snakemake/scripts/.snakemake.qdau8yc6.metrics.py ' returned non-zero exit status 1.
  File "/home/dewald/Documents/snakemake/Snakefile", line 86, in __rule_metrics
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Will exit after finishing currently running jobs.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2021-03-17T150010.750793.snakemake.log
